metagpt/utils/omniparse_client.py

```diff
@@ -0,0 +1,239 @@
+import mimetypes
+import os
+from pathlib import Path
+from typing import Union
+
+import httpx
+
+from metagpt.rag.schema import OmniParsedResult
+from metagpt.utils.common import aread_bin
+
+
+class OmniParseClient:
```

Comment:
Here already has a `omniparse-client`, should we use it directly.

Reply:
`OmniParseClient` can be used alone as an independent client (SDK) of omniparse service, while the encapsulated `OmniParse` is integrated into `llamaindex` to meet its data parsing specifications, but it maintains an `OmniParseClient` internally for real document parsing, so that the encapsulation is more decoupled